lib/net/peer.js

```diff
@@ -496,6 +498,11 @@ class Peer extends EventEmitter {
     this.invTimer = setInterval(() => {
       this.flushInv();
     }, Peer.INV_INTERVAL);
+
```

Comment:
It makes more sense to initialize the `SlidingWindow` here compared to the `constructor` so that its only initialized if there is a successful p2p handshake. :+1:

Reply:
:+1: